<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1546781733986' stop='1546782131254'>
    <name>Sample test case</name>
    <title>Sample test case</title>
    <test-cases>
        <test-case start='1546781733986' status='failed' stop='1546782131253'>
            <name>Login</name>
            <title>Login</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='b0be54f2-aad0-409b-92e4-6482cf64a262-attachment.png' type='image/png' size='173591'/>
            </attachments>
            <failure>
                <message>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.</message>
                <stack-trace>Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
                    at Timeout._onTimeout (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4281:23)
                    at ontimeout (timers.js:436:11)
                    at tryOnTimeout (timers.js:300:5)
                    at listOnTimeout (timers.js:263:5)
                    at Timer.processTimers (timers.js:223:10)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>